next.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  images: {
+    domains: [
+      'localhost',
+      '35.182.254.224',
+      '35.182.99.194',
```

Comment:
Whats the difference between the two 35.xxx.... domains? just curious

Reply:
It is just two different server IP. 

We initially launched one by it isn't enough power so we made another one and that got assigned a different public IP.

Reply:
What do you mean by "power"? what resources does it use? like cpu and memory?

Reply:
strapi required that the vm has more than 1gb of ram lol. We chose the one with just 1gb at first